src/Advisor/Advisor.Autorest/README.md

```diff
@@ -43,15 +43,6 @@ module-version: 0.1.0
 # Normally, title is the service name
 title: Advisor
 subject-prefix: $(service-name)
-resourcegroup-append: true
-
-# If there are post APIs for some kinds of actions in the RP, you may need to 
-# uncomment following line to support viaIdentity for these post APIs
-# identity-correction-for-post: true
-
-# For new modules, please avoid setting 3.x using the use-extension method and instead, use 4.x as the default option
-use-extension:
-  "@autorest/powershell": "3.x"
 
 directive:
   # Following is two common directive which are normally required in all the RPs
```

Comment:
Please follow https://github.com/Azure/autorest.powershell/blob/main/docs/migration-from-v3-to-v4.md#to-match-variants to update the directive from line#60~62